### DISCUSSION TITLE
LaserGrbl versus Lightburn

### DISCUSSION THREAD

**Discussion by nonpds** [arkypita/LaserGRBL#1425] [Q&A]
I use both LaserGrbl and Lightburn software, I prefer LaserGrbl but have noticed that Lightburn will cut 3mm ply in 3 passes but it takes 6 or 7 much slower passes to do the same with LaserGrbl. Anyone have any ideas why this might be? I am really confused.

**Comment by arkypita** [comment]
Do you use M3 or M4? Did you read lasergrbl faq?
https://lasergrbl.com/faq

**Comment by alastaira** [comment]
Ultimately, all LaserGRBL and Lightburn do is just create a series of gcode commands and send them to your laser controller to execute.
If you're noticing a difference in behaviour between the two applications, it's because they're generating different gcode instructions. Create the simplest test case you can that demonstrates the issue (i.e. a 10x10mm square?) and compare the output (or post it here)
